src/helper/task.ts

```diff
@@ -265,10 +265,27 @@ export class ZkWasmServiceHelper {
       console.log("json", queryJson);
     }
 
-    let tasks = await this.endpoint.invokeRequest("GET", `/tasks`, queryJson);
+    let tasks = (await this.endpoint.invokeRequest(
+      "GET",
+      `/tasks`,
+      queryJson
+    )) as PaginationResult<Task[]>;
     if (this.endpoint.enable_logs) {
       console.log("loading task board!");
     }
+    // Convert the corresponding proof fields into Uint8Array from number[] (json)
```

Comment:
Better switch this comments with 277 line so it is more clear for the below converting codes